tools/chpldoc/chpldoc.cpp

```diff
@@ -393,6 +389,14 @@ static bool isNoWhereDoc(const Function* f) {
   return false;
 }
 
+static bool isHideImplType(const Decl* e) {
+  if (auto attrs = e->attributeGroup()) {
```

Comment:
Can we skip the check for the attribute group?

Reply:
OK, I figured that the check would be occurring in `ast->hasAttribute` (don't we have to have an attribute group to get the attribute?).

Reply:
I think we have `Ast::hasPragma`, at least I see it in `chpl/uast/AstNode.h`. I think it would be nice to have `hasAttribute`. And for the rest of the properties I think we should add them on a priority/usage basis.